api/controllers/agreements-controller.js

```diff
@@ -448,11 +417,9 @@ const createAgreement = asyncMiddleware(async (req, res) => {
   //     agreement.hoardAddress = ref.address;
   //     agreement.hoardSecret = ref.secretKey;
 
-  //     log.debug('Request to create new agreement: ' + agreement.name);
-
   const agreementAddress = await contracts.createAgreement(agreement);
   await contracts.setMaxNumberOfEvents(agreementAddress, parseInt(req.body.maxNumberOfEvents, 10));
-  await contracts.updateAgreementEventLog(agreementAddress, hoardRef);
+  await contracts.updateAgreementEventLog(agreementAddress, JSON.stringify(''));
```

Comment:
Do we need to call it with an empty reference, if it's automatically created via the endpoint which adds external references? It looks like we can remove this call.